IDR/production/2.visualize_stats.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+from plotnine import *
+import pathlib
+from sigfig import round
+
+
+if __name__ == "__main__":
+
+    # Load data
+    databank_stats_dir = pathlib.Path("IDR/data/statistics/databank_diversity.parquet")
+    databank_stats = pd.read_parquet(databank_stats_dir)
+
+    study_stats_dir = pathlib.Path(
+        "IDR/data/statistics/individual_studies_diversity.parquet"
+    )
+    study_stats = pd.read_parquet(study_stats_dir)
+
+    dodge_text = position_dodge(width=0.9)
+
+    # Round to 3 sigfigs if not 0
+    stats_to_round = ["H", "J", "NME", "E", "GC"]
+    for stat in stats_to_round:
+        databank_stats[stat] = databank_stats[stat].apply(
+            lambda x: round(x, sigfigs=3) if x > 0 else x
+        )
+        study_stats[stat] = study_stats[stat].apply(
+            lambda x: round(x, sigfigs=3) if x > 0 else x
+        )
+
+    # Set output images directory
+    study_imgs_dir = pathlib.Path("IDR/data/statistics/imgs/study_stat_imgs")
+    databank_imgs_dir = pathlib.Path("IDR/data/statistics/imgs/databank_stat_imgs")
+
+    # Plot stats for databank and individual study stats
+    stats_to_graph = ["H", "J", "NME", "E", "GC", "S"]
```

Comment:
I think the figures would be improved if these labels were spelled out. Would this be an easy fix or a pain?

Reply:
Got it. I just added a dictionary with titles that correspond to each statistic abbreviation. Easy!

Added in ef55c120b71ad2b684532cb3db620e1f97a0bcc8.